frontend/src/pages/dashboard/components/sidebar/styles.module.scss

```diff
@@ -22,28 +23,29 @@
   position: relative;
   width: 100%;
   height: 100%;
+  margin-bottom: 9px;
   padding: 16px 0;
-  color: $black-light;
+  color: $grey;
   font-weight: $font-weight-bold;
-  text-transform: capitalize;
+  border-radius: 8px;
   box-shadow: none;
 }
 
+.item:first-child::after {
```

Comment:
I think this line should be under the active tab, not just the first tab?

Reply:
yeap, refactor it